course/models/models.py

```diff
@@ -185,6 +185,19 @@ def has_view_permission(self, user):
     def has_edit_permission(self, user):
         return user.is_teacher
 
+    def clone(self, course, event, author):
+        question_clone = self
```

Comment:
This will modify the current instance I think!
You can confirm it by getting a question and then clone it and check if the original one is changed or not.
Please check if it modifies the original one you have to change this to get a copy of `self`, not the `self` itself.

Reply:
I have tested and this does not modify the current instance. Once the `id` and respective fields are set to `None`, a new instance is created and the new instance is the one being modified.

Reply:
<img width="403" alt="question clone" src="https://user-images.githubusercontent.com/22093299/126883618-adf4882f-9f29-4592-b5fc-36089eb97124.png">

Cloning from q1 modifies the q1 but as long as it's not saved the changes won't appear in the database.

Reply:
I've attached a video. As you can see in the demo the original questions and events are not modified with the current code when importing an event.
https://user-images.githubusercontent.com/59884796/127220138-2b0ea408-e81b-4de0-beac-eca4688719a8.mp4